src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java

```diff
@@ -74,4 +80,18 @@ public FinancialTransactionCategory(String name, FinancialTransactionType type)
         this.creationDate = Instant.now();
     }
 
+    public FinancialTransactionCategory(String name, FinancialTransactionType type, User user) {
+        this.name = name;
+        this.type = type;
+        this.creationDate = Instant.now();
+        this.user = user;
+    }
+
+    public void addUser(User user){
+        this.setUser(user);
+    }
+
+    public void removeUser(){
+        this.setUser(null);
+    }
 }
```

Comment:
Please remove lines from 90 - 96 and add these one: 
    public void addFinancialTransactionCategory(FinancialTransactionCategory financialTransactionCategory){
        this.financialTransactionCategories.add(financialTransactionCategory);
        financialTransactionCategory.setUser(this);
    }

    public void removeFinancialTransactionCategory(FinancialTransactionCategory financialTransactionCategory) {
        this.financialTransactionCategories.remove(financialTransactionCategory);
        financialTransactionCategory.setUser(null);
    }

Reply:
Added this method to User class, according to our findings.